plugins/dynamic-component.ts

```
@@ -0,0 +1,110 @@
+  const entrypoints = (
+    await Promise.all(
+      dynamicPaths?.map((path) =>
+        Array.fromAsync(
+          glob.scan({
+            cwd: resolve(cwd, path),
+            absolute: true,
+          })
+        )
+      )
+    )
+  ).reduce((p, n) => [...p, ...n], []);
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)